ui/perfherder/graphs/GraphsContainer.jsx

```diff
@@ -332,6 +342,16 @@ class GraphsContainer extends React.Component {
       width,
     } = this.state;
 
+    const affectedData = [];
```

Comment:
This name isn't too clear... A reader's first question would be "Affected by what?".